ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js

```diff
@@ -55,8 +56,38 @@
                 this.options.queryModel.set('type', 'text');
                 user.getQuerySettings().set('type', 'text');
                 break;
+            case 'newResult':
+                oldType = this.options.queryModel.get('type');
+                this.options.queryModel.set({
+                    type: 'newResult',
+                    title: "",
+                    modelId: "",
+                    accessGroups: [],
+                    accessIndividuals: [],
+                    descriptors: [],
+                    description: ""
+                });
+                if (oldType  === 'newResult') {
```

Comment:
Should we revisit extracting this `if` block from each case statement to make it more generic? Seems repetitious to have it called so many times.